pkg/util/command_util.go

```diff
@@ -334,41 +335,33 @@ func GetUIDGidFromUserString(commandUserStr string, replacementEnvs []string) (s
 }
 
 func GetUserFromUsername(userStr string, groupStr string) (string, string, error) {
-	// Lookup by username
-	userObj, err := user.Lookup(userStr)
+	// Default to userStr if it is a numeric value like "1000"
+	uid := userStr
+	// Check to see if userStr is numeric ID
+	_, err := strconv.ParseUint(userStr, 10, 32)
 	if err != nil {
-		if _, ok := err.(user.UnknownUserError); ok {
-			// Lookup by id
-			userObj, err = user.LookupId(userStr)
-			if err != nil {
-				return "", "", err
-			}
-		} else {
+		// Was not numeric, try lookup by username
+		userObj, err := user.Lookup(userStr)
+		if err != nil {
+			logrus.Errorf("User string cannot be found and is not numeric %s", userStr)
 			return "", "", err
 		}
+		uid = userObj.Uid
 	}
 
 	// Same dance with groups
-	var group *user.Group
+	gid := ""
```

Comment:
Shouldn't this start as the `groupStr` because when you successfully parse a number you aren't setting `gid` again